plugins/cue_points/quiz/QuizPlugin.php

```diff
@@ -575,33 +592,45 @@ protected function getAggregateDataForQuestions($questions, $orderBy)
 			$c->add(CuePointPeer::TYPE, QuizPlugin::getCoreValue('CuePointType', QuizCuePointType::QUIZ_ANSWER));
 			$c->add(CuePointPeer::PARENT_ID, $question->getId());
 			$answers = CuePointPeer::doSelect($c);
-			$numOfAnswers = count($answers);
-			if ($numOfAnswers)
+			$numOfAnswers = 0;count($answers);
+			$quizUserEntries = array();
+			foreach ($answers as $answer)
 			{
-				foreach ($answers as $answer)
+				/**
+				 * @var AnswerCuePoint $answer
+				 */
+				$quizUserEntryId = $answer->getQuizUserEntryId();
```

Comment:
Arange this code same as in getAnswerCountByUserIds.
For better reading remove the local caching.